packages/meditrak-server/src/index.js

```diff
@@ -8,7 +8,7 @@ import '@babel/polyfill';
 import {} from 'dotenv/config'; // Load the environment variables into process.env
 
 import http from 'http';
-import { TupaiaDatabase, ModelRegistry } from '@tupaia/database';
+import { TupaiaDatabase, ModelRegistry, EntityHierarchyCacher } from '@tupaia/database';
```

Comment:
meditrak-server is responsible for listening to and responding to all other changes, so I decided it should also be where the entity hierarchy cache is invalidated and rebuilt